R/AllClassUnions.R

```diff
@@ -1,2 +1,3 @@
 setClassUnion("mzIDClasses", c("mzID", "mzIDCollection"))
 setClassUnion("logicalOrNumeric", c("logical", "numeric"))
+setClassUnion("characterOrFunction", c("character", "function"))
```

Comment:
Do we really need this class union? The user never use `ProcessingStep` directly, or? And we could stick with one method (you used `character` before, any use case for `function`?)

Reply:
I would like to keep this version. I want to be able to define also a `ProcessingStep` like `ProcessingStep(FUN = function(z, t) z * t, ARGS = list(t = 5))`, i.e. pass along functions, not only function names (provided as `character`).

Reply:
Ok, fine.